rollup-node.md

```diff
@@ -0,0 +1,285 @@
+# Rollup Node Specification
+
+<!-- All glossary references in this file. -->
+[rollup node]: /glossary.md#rollup-node
+[derivation]: /glossary.md#L2-chain-derivation
+[L2 derivation inputs]: /glossary.md#L2-derivation-inputs
+[payload attributes]: /glossary.md#payload-attributes
+[block]: /glossary.md#block
+[execution engine]: /glossary.md#execution-engine
+[reorg]: /glossary.md#re-organization
+[block gossip]: /glossary.md#block-gossip
+[rollup driver]: /glossary.md#rollup-driver
+[deposits]: /glossary.md#deposits
+[deposit-feed]: /glossary.md#L2-deposit-feed-contract
+[L2 chain inception]: /glossary.md#L2-chain-inception
+
+The [rollup node] is the component responsible for [deriving the L2
+chain][derivation] from L1 blocks. This process happens in two steps:
+
+1. Read [L2 derivation inputs] from L1 and generate [payload attributes] (essentially
+   [a block without output properties][block]).
+2. Pass the payload attributes to the [execution engine], so that [output block
+   properties][block] may be computed.
+
+While this process is conceptually a pure function from the L1 chain to the L2
+chain, it is in practice incremental. The L2 chain changes whenever new L1
+blocks (and hence new L2 derivation inputs) are added to the L1 chain, or
+whenever the L1 chain [re-organizes][reorg].
+
+Additionally, the rollup node is responsible for [block gossip], i.e.
+transmitting the L2 blocks it derives over a peer-to-peer network.
```

Comment:
Same as other comment about p2p/transmitting: it is the job of the sequencer to seed the gossip, and unrelated to local derivation at the start, until the node figures out it either derives the same thing it already propagated, or derived something that will reorg away the previous sequencer claim.